docs/components/stepper.md

```diff
@@ -0,0 +1,130 @@
+# Stepper
+
+**KStepper** - A basic Stepper component
+
+<KStepper :steps="defaultItems" />
+
+```html
+<KStepper :steps="steps" />
+```
+
+## Props
+
+### steps
+
+An array of `KSteps`. Each `KStep` has a `label` and an optional `state`.
```

Comment:
Since we don't expose the usage of the `KStep` component, I think we should update the references in this file to just talk about steps as objects within an array instead of referring to the internal-only component
```suggestion
An array of step objects. Each step object should have a required `label` property, and an optional `state` property.
```